.eslintrc.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+    env: {
+        browser: true,
+        es2020: true,
+    },
+    extends: [
+        'airbnb-typescript',
+    ],
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        ecmaVersion: 11,
+        sourceType: 'module',
+        project: './tsconfig.json',
+    },
+    plugins: [
+        '@typescript-eslint',
+    ],
+    rules: {
+        indent: ['error', 4],
+        '@typescript-eslint/naming-convention': [
+            'error',
+            {
+                selector: 'variableLike',
+                format: ['camelCase', 'UPPER_CASE', 'PascalCase'],
+            },
+        ],
+        '@typescript-eslint/indent': ['error', 4],
+        'import/first': ['off'],
+        'import/order': ['off'],
+        'import/prefer-default-export': ['off'],
+        'max-len': ['error', 120],
+        'object-curly-newline': ['off'],
```

Comment:
Configuration has been changed slightly to make this fit the project slightly better. These changes may be needed across the project eventually but are fine for the time being

Reply:
Are we set on using the airbnb style guide?
I think its a good move but just to confirm.

Reply:
Not set on it, I tend to just use it as a base to work from and then tweak it. Haven't looked too much into the other options but Google and Standard are the other two popular ones I think?

Reply:
I'm fine just going with the airbnb style for now, we can change as needed later.